src/figuresClasses.ts

```diff
@@ -1,19 +1,87 @@
-export interface Figure {
+type Colors = 'red' | 'green' | 'blue';
 
+enum Shape {
+  Triangle = 'triangle',
+  Rectangle = 'rectangle',
+  Circle = 'circle',
 }
 
-export class Triangle {
-
+export interface Figure {
+  shape: Shape;
+  color: Colors;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape = Shape.Triangle;
+
+  constructor(
+    public color: Colors,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [a, b, c];
+
+    sides.sort((x, y) => y - x);
+
+    if (sides[0] >= sides[1] + sides[2]) {
```

Comment:
please use desctructured value of sides
```suggestion
    if (a >= b + c) {
```